### PR TITLE
Add Code to Valitated Contact Form

### PR DESCRIPTION
**Pull request by Fernando426**
- Added JS, HTML, CSS code for contact form validation
- JavaScript code has been added for a simple contact form validation on my portfolio website. The validateInputs() function is responsible for checking if the email field meets certain conditions before submitting the form.

The validation ensures that the value entered in the email field is in lowercase. If the email field contains any uppercase letters, it is considered invalid, and a CSS class called "invalidEmail" is added to the element that displays the validation message.

The form is associated with a submit event, which means that when the user tries to submit the form, the validateInputs() function is executed first. If the function returns true (indicating that the email is valid), then the form is submitted through the form.submit() call.

In summary, this code provides basic validation to ensure that the email field is in lowercase before submitting the contact form on a web page.

### CODE REVIEW

**Review by Whoistolu — APPROVED** [review]
### Hi @Fernando426 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Highlight
🔆 The contact form contains validation checking if the email is written in all lowercase
🔆 The error message shows up close to the `submit` button

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
